apps/studio/components/interfaces/Settings/Infrastructure/UpgradeWarnings.tsx

```diff
@@ -67,12 +70,17 @@ export const UnsupportedExtensionsWarning = ({
       <AlertTitle_Shadcn_>A new version of Postgres is available</AlertTitle_Shadcn_>
       <AlertDescription_Shadcn_ className="flex flex-col gap-3">
         <div>
-          <p className="mb-1">You'll need to remove the following extensions before upgrading:</p>
+          <p className="mb-1">The following extensions have to be removed before upgrading:</p>
 
           <ul className="pl-4">
             {unsupportedExtensions.map((obj: string) => (
               <li className="list-disc" key={obj}>
-                {obj}
+                <a
```

Comment:
Why are you using `<a>` here? Were you planning to add `target="_blank"`?

Reply:
oh 🤦  should be using Link here sorry!

Reply:
just pushed! 🙏